### ISSUE TITLE
Table: This th element should not be empty.

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3849]
Description: This th element should not be empty.

![image](https://user-images.githubusercontent.com/32538291/98776811-acfb5e00-2415-11eb-9cf5-a7c9b37f2627.png)

Expected: All table header cells must specify text content or alternative text on an image within table header cell (th). An exception to this requirement is for the first cell in a table when a table contains row and column headers and no header cell is needed to -- in this case a td cell should be used on the empty cell.

Screenshot:

![image](https://user-images.githubusercontent.com/32538291/98776842-bab0e380-2415-11eb-8c4d-75b78fea070b.png)
